sites/main-page-title/includes/main-page-title.html

```diff
@@ -0,0 +1,12 @@
+{% if include %}
```

Comment:
Is that needed? This include files are going to be copied inside our gcweb-jekyll `_include` folder.

```suggestion
```